app/javascript/app/components/my-climate-watch/viz-creator/viz-creator-selectors.js

```diff
@@ -33,6 +33,7 @@ export const categoriesSelector = state => get(lenses.$categories, state);
 export const subcategoriesSelector = state => get(lenses.$subcategories, state);
 export const yearsSelector = state => get(lenses.$years, state);
 export const timeseriesSelector = state => get(lenses.$timeseries, state);
+export const getTitle = state => state.title || null;
```

Comment:
Maybe `titleSelector` would be a better name to follow the naming convention of the other selectors, what do you think?